styles/variables-base.less

```diff
@@ -363,19 +363,33 @@
 @agate-spinner-size: inherit;
 @agate-spinner-bg-margin: inherit;
 @agate-spinner-bg-padding: inherit;
+@agate-spinner-huge-size: inherit;
```

Comment:
It may be a good idea to start adding these variables' relationships and default sizes into this file now, to save a bit of work here and in the future. Thoughts?